ws-nextjs-app/pages/[service]/live/[id]/ShareButton/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { Theme, css } from '@emotion/react';
+import pixelsToRem from '../../../../../../src/app/utilities/pixelsToRem';
+
+export default ({ palette, fontSizes, fontVariants, spacings, mq }: Theme) =>
+  css({
+    color: palette.BLACK,
+    ...fontSizes.pica,
+    ...fontVariants.sansBold,
+    padding: `${spacings.FULL + spacings.HALF}rem`,
+    marginTop: `${spacings.FULL}rem`,
+    marginBottom: `${spacings.TRIPLE}rem`,
+    marginInlineStart: `${spacings.DOUBLE}rem`,
+    border: `${pixelsToRem(2)}rem solid ${palette.BLACK}`,
+    backgroundColor: 'transparent',
+    cursor: 'pointer',
+    '&:hover, &:focus-visible': {
+      border: `${pixelsToRem(2)}rem solid ${palette.BRAND_BACKGROUND}`,
```

Comment:
The focus outline doesn't work in Firefox (I see a dotted line), are we able to use our custom classes we already have for adding the focus indicator?

Reply:
This isn't to do with this styling, looks like firefox has an override style for type=button. I think we'll need to add type='button' to the [custom classes](https://github.com/bbc/simorgh/blob/64f64fa61991713cbad3cbcb64c6ef1ec876b799/src/app/components/ThemeProvider/focusIndicator.ts#L10)

![image](https://github.com/bbc/simorgh/assets/4706195/5bccbe1e-9d85-4399-9c3a-d14aef8164a3)

Reply:
Or we can remove type button, don't think it's strictly needed here.

Reply:
It's needed as browsers would treat it as a submit button for a form https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/button-has-type.md

Reply:
Believe that is only if it's wrapped inside a form tag... but cool

Reply:
Apparently if you don't add it, it's possible that it can destroy the current state of the document https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#notes

Reply:
I've done this now